doc/api/fs.md

```diff
@@ -7509,6 +7509,18 @@ added:
 
 Free blocks available to unprivileged users.
 
+```mjs
+import { statfs } from "fs/promises";
+
+async function getAvailableSpace(path) {
```

Comment:
For the sake of simplicity, isn't it better to use top level await instead of an async function?